js/components/detailsItem/detailsItem_directive.js

```diff
@@ -8,6 +8,7 @@ angular.module('contactsApp')
 			name: '=',
 			data: '=',
 			contact: '=model',
+			addressBook: '=addressBook',
```

Comment:
Just FYI this line is to specific: `addressBook: '='` would suffice.
You only need to write something after the `=` if the attribute should be named differently to the controller variable.